public/index.html

```diff
@@ -1,27 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-<title>Rock, Paper, Scissors</title>
-<meta charset="UTF-8">
+    <title>Rock, Paper, Scissors</title>
+    <meta charset="UTF-8">
+    <script src="simplepeer.js"></script>
```

Comment:
maybe put this script at the end of the file where all the other scripts are included

Reply:
done